tests/python/gaia-ui-tests/gaiatest/apps/ring_tone/app.py

```diff
@@ -8,14 +8,17 @@
 
 
 class RingTone(Base):
-    name = 'Ringtones'
+    origin = 'app://ringtones.gaiamobile.org'
+    _page_locator = (By.ID, 'list-parent')
+    _header_locator = (By.ID, 'header')
     _ring_tone_locator = (By.CSS_SELECTOR, '#list-parent section > ul > li')
     _set_button_locator = (By.ID, 'set')
     _save_button_locator = (By.ID, 'save')
+    _actions_cancel_locator = (By.CSS_SELECTOR, '[data-l10n-id="actions-cancel"]')
 
     def __init__(self, marionette):
         Base.__init__(self, marionette)
-        Wait(self.marionette).until(lambda m: self.apps.displayed_app.name == self.name)
+        Wait(self.marionette).until(lambda m: self.apps.displayed_app.origin == self.origin)
```

Comment:
Cool! I think that's exactly what needs to be done to make this working in other locales. I thought of doing that in bug 1188926, but good that you're already changing this!